netmaster/master/netmaster.go

```diff
@@ -53,6 +54,7 @@ func SetClusterMode(cm string) error {
 	}
 
 	masterRTCfg.clusterMode = cm
+	SetDNSEnabled(false)
```

Comment:
Hmm, This is probably not correct. If i run netmaster -cluster-mode docker, this will disable DNS.
Plus, It'll be good to keep all flag parsing/setting logic in parseOpts() or execOpts() functions..